CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ The REST API versions in the table are just for your information as the values a
 - Ranking filters that show the data from a set of the highest or lowest ranked values of an attribute based on the ranking criteria that you establish ([doc](https://sdk.gooddata.com/gooddata-ui/docs/ranking_filter_component.html))
 - The option to turning off data points in line charts, area charts, and combo charts ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-canvases))
 
+## 7.9.0
+
+### Added
+- Upgrade dependencies to their latest versions possible without breaking compatibility.
```

Comment:
The dependencies are upgraded to their latest possible versions to still be backward-compatible.

Reply:
Updated, thanks 🙂